### DISCUSSION TITLE
Channel to ACINQ node disabled on ACINQ's side since opening.

### DISCUSSION THREAD

**Discussion by ghost** [ACINQ/eclair#2778] [Q&A]
I've opened a channel to the ACINQ node yesterday and it has been disabled on ACINQ's side since then.

I am using LND as Lightning Network implementation and your node is the only one I've ever had this issue with. Not sure what to look for. Here is the output of `lncli getchaninfo 897876588463783937`:

```
{
    "channel_id":  "897876588463783937",
    "chan_point":  "903e0e416888f2cd5c48dd5eddbad8e95250f56523c2cbbd8d17820fc9704f98:1",
    "last_update":  1699982255,
    "node1_pub":  "03864ef025fde8fb587d989186ce6a4a186895ee44a926bfc370e2c366597a3f8f",
    "node2_pub":  "03de3ae39bf77002092456c93c43652fe6fdbc455b24966c62d8afb6a0c3be0ef1",
    "capacity":  "4000000",
    "node1_policy":  {
        "time_lock_delta":  144,
        "min_htlc":  "1",
        "fee_base_msat":  "1000",
        "fee_rate_milli_msat":  "499",
        "disabled":  true,          <---------------- Disabled on your end.
        "max_htlc_msat":  "1800000000",
        "last_update":  1699890241,
        "custom_records":  {}
    },
    "node2_policy":  {
        "time_lock_delta":  144,
        "min_htlc":  "1000",
        "fee_base_msat":  "0",
        "fee_rate_milli_msat":  "2475",
        "disabled":  false,         <---------------- Enabled on my end.
        "max_htlc_msat":  "1800000000",
        "last_update":  1699982255,
        "custom_records":  {}
    },
    "custom_records":  {}
}

``` 
Any clarification will be appreciated.

**Comment by t-bast** [comment]
That is expected behavior, why is that an issue? All the liquidity is on your side, there is no way to route payments from our node to yours (since there is no liquidity available there), that's why the channel is initially disabled in that direction. Once you relay outgoing payments, that will push liquidity to our side, and we will emit a new `channel_update` that enables that direction.

**Reply by ghost** [reply]
I thought this could prevent me from rebalancing channels by sending sats out from the ACINQ channel and in another one.

**Reply by t-bast** [reply]
Not at all, it doesn't, don't worry.

**Comment by t-bast** [comment]
Note that we've changed this behavior recently though: we will start advertising that there's no liquidity on our side by using a low `max_htlc_amount_msat` instead of using the `disabled` flag (which will then be reserved for cases where the other node is offline).

**Comment by ghost** [comment]
Thank you very much for the explanation.
